OmiseSwift/Models/Digits.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+
+public struct Digits: Equatable {
+    public let digits: String
+    
+    private static let invalidCharacterSet = CharacterSet(charactersIn: "0123456789").inverted
```

Comment:
Aren't these actually valid characters? 🤔

Reply:
It's an inverted of the character in the list :)